lib/task_helper.py

```diff
@@ -0,0 +1,40 @@
+import json, sys
```

Comment:
Not sure how much value using pylint/autopep8 would add here. But things like multiple imports on one line and some of the spacing (like no space between parameter and default value in method definitions) are PEP8 violations.